src/main/java/medicaldoctor/controllers/navbar/patientRegistrationController.java

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
you are still not removing these. once again just because it is autogenerated doesn't mean you are forced to keep it... go through all your files and clean them up.

Reply:
with autoformatting too